llvm/include/llvm/Passes/MachinePassRegistry.def

```diff
@@ -97,6 +97,7 @@ LOOP_PASS("loop-term-fold", LoopTermFoldPass())
 // LiveVariables can be removed completely, and LiveIntervals can be directly
 // computed. (We still either need to regenerate kill flags after regalloc, or
 // preferably fix the scavenger to not depend on them).
+MACHINE_FUNCTION_ANALYSIS("edge-bundles", EdgeBundlesAnalysis())
```

Comment:
May need a pinter pass call the `view` method to dump egds.

Reply:
Can introduce in a new patch